var/spack/repos/builtin/packages/python/package.py

```diff
@@ -190,7 +190,7 @@ def setup_build_environment(self, env):
         # Python v2.7 and v3.4+ (see https://bugs.python.org/issue1180) and
         # adding support for ignoring user configuration will require
         # significant changes to this package for other Python versions.
-        if not spec.satisfies('@2.7,3.4:'):
+        if not spec.satisfies('@2.7:2.8,3.4:'):
```

Comment:
I'm not sure whether or not this actually works with the current concretizer. There have been longstanding issues with shortened versions (see #921). Staying on the safe side here.